server/api/api.go

```diff
@@ -96,6 +101,10 @@ func (api *api) MakeClient() (remote.Client, error) {
 	return api.Remote.NewClient(context.Background(), api.user.OAuth2Token), nil
 }
 
+func (api *api) MakeAppClient() (remote.Client, error) {
```

Comment:
0/5 somehow I find `App` to be insufficiently intention-revealing here (same in Jira, ``MakeClientForServer` or something? - no good). `MakeSuperuserClient` or at least `MakeAppLevelClient` to be consistent with `remote`.

Also 0/5 suggest `s/Make/New/`, and removing the unused error return?